pkg/runtimemetrics/tags.go

```diff
@@ -0,0 +1,69 @@
+package runtimemetrics
+
+import (
+	"fmt"
+	"math"
+	"runtime/metrics"
+)
+
+const gogcMetricName = "/gc/gogc:percent"
+const gomemlimitMetricName = "/gc/gomemlimit:bytes"
+const gomaxProcsMetricName = "/sched/gomaxprocs:threads"
+
+func getBaseTags() []string {
+	baseTags := make([]string, 0, 3)
```

Comment:
NIT: We could move this below and do:

```suggestion
	baseTags := make([]string, 0, len(samples))
```